docs/beanstalk/03_Setting_Up_Mongo/README.md

```diff
@@ -0,0 +1,157 @@
+At this point, your MongoDB server will be accessible from your current IP address and the Elastic Beanstalk server. If you are using a VPN or public IP, remember to remove access as soon as you are done.
+
+## Constructing the MongoDB Connection String
+
+Now that we have the database credentials, the private and public ip address of your MongoDB server, you can create the connection strings. Use the following template to construct your connection strings, one will be used to access the database from your local machine and the other from the EB server.
+
+Public Connection String:
+```
+mongodb://<username>:<password>@<publicIP>/admin
+```
+
+Private Connection String (used by your backend server):
+```
+mongodb://<username>:<password>@<privateIp>/cscl?authSource=admin
+```
+*Note that "cscl" in the Private connection string is the name of your database. Update the connection string accordingly if you are using a different database.*
+
+## Connecting to MongoDB Using MongoDB Compass and Importing Your Data
+Now that you have network access to the database, you can use MongoDB Compass to connect to it and import your data.
+
+<br>
+1)  In MongoDB Compass click "New Connection" and enter the "Public Connection String" you created earlier in the text field then click "CONNECT".
+
+![MongoDB Compass connection creation screen with Public Connection String](./images/Mongo_Compass/1_Mongo_Compass_connect_to_server.png)
+
+
+<br>
+2) Once logged in you will be presented with the current databases on your MongoDB server. Click the "CREATE DATABASE" button at the top to create a new database.
+
+![MongoDB Compass Databases screen listing all current databases](./images/Mongo_Compass/2_Mongo_Compass_create_db_screen.png)
+
+
+<br>
+3) On this form, enter the desired database name. For these instructions "cscl" will be used for the database. "items" will be used for the collection name. Click "Create Database" once done. 
+
+![MongoDB Compass create database form with "csc" database and "items" collection](./images/Mongo_Compass/3_Mongo_Compass_creating_database_and_collection.png)
+
+
+<br>
+4) Once your database is created you will be presented with the list of all databases. Select the database you created, i.e. "cscl". 
+
+![MongoDB Compass databases list with "cscl" highligthed](./images/Mongo_Compass/4_Mongo_Compass_select_database.png)
+
+
+<br>
+5) Select the collection that you created under your database, i.e. "items"
+
+![MongoDB Compass collections list for "cscl" database with "items" highlighted](./images/Mongo_Compass/5_Mongo_Compass_select_collection.png)
+
+
+<br>
+6) Click the "ADD DATA" button in the upper-left, and select "Import File".
+
+![MongoDB](./images/Mongo_Compass/6_Mongo_Compass_import_file_button.png)
+
+
+<br>
+7) Select the JSON file provided to you in the [template repository](https://github.com/Hack-Diversity/template-mini-project-db) or your own MongoDB backup. Enable the Options that you would like, and click "IMPORT".
```

Comment:
Missing the bang in front of `![template repository]` which is preventing the link from getting created correctly ☹️